how-to/customize-workspace/package.json

```diff
@@ -20,7 +20,8 @@
 		"setup": "cd ../../ && npm install && cd how-to/customize-workspace && npm run build && npm --prefix ../common/ run build",
 		"setup-dev": "npm install concurrently live-server --save-dev && npm run build-server-live",
 		"validate": "npm --prefix ../../ run validate",
-		"generate-types": "tsc --project ./client/tsconfig.types.json"
+		"generate-types": "tsc --project ./client/tsconfig.types.json",
+		"package-content": "node ./scripts/package-content.mjs"
```

Comment:
it looks like the pacakge.json is missing the dev dependencies in case they are installed locally.

Reply:
Add fast glob to devdeps